{"name": "test_case3_", "status": "failed", "statusDetails": {"message": "AssertionError: assert (1 + 3) == 13", "trace": "def test_case3_():\n>       assert 1+3==13\nE       assert (1 + 3) == 13\n\ntestdemo.py:8: AssertionError"}, "start": 1699593620052, "stop": 1699593620052, "uuid": "50531049-6687-4547-8b56-90e95084259b", "historyId": "8742959524cd8d4629e5bcc19bed72d2", "testCaseId": "8742959524cd8d4629e5bcc19bed72d2", "fullName": "testdemo#test_case3_", "labels": [{"name": "suite", "value": "testdemo"}, {"name": "host", "value": "LAPTOP-6Q1JAI9B"}, {"name": "thread", "value": "105556-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testdemo"}]}